### DISCUSSION TITLE
Size (capacity) of micro SD cards

### DISCUSSION THREAD

**Discussion by gerdb** [cardonabits/haxo-hw#38] [General]
The size of the provided images is 16GB: or 15.931.539.456 bytes. The SD card must have at least this size.

My I**ntenso 16GB MicroSD (1) HCI Class 10** has only 15.634.268.160 bytes. Too less.
My **SanDisk Ultra 16GB micro HC I 10 A1** has exact the same size as the images: 15.931.539.456 bytes
It worked with this SanDisk 16GB SD card or with one with 32GB.

I successfully tried [PiShrink](https://github.com/Drewsif/PiShrink) with the option -s which shrinks the 16GB down to 4.5GB.

**Comment by jcard0na** [comment]
Nice tip.  I had tried PiShrink as well, but encountered [this issue](https://github.com/Drewsif/PiShrink/issues/80).  This means that once an image was (Pi)shrunk once, it cannot be shrunk a second time.  This makes it inconvenient if you want to make changes to the image and re-shrink it for distribution.
